webpack.config.js

```diff
@@ -21,6 +21,10 @@ module.exports = {
         libraryTarget: 'umd',
         library: 'OSG'
     },
+    node: {
+       fs: "empty"
+    },
+    target: 'node',
```

Comment:
Not sure if it impacts other code, see https://webpack.js.org/concepts/targets/ for info. I did this to be able to access to filesystem while testing in node. 

Maybe @arthurjamain can shed some light.